src/main/java/lesson03/conditions/LuckyTicket.java

```diff
@@ -0,0 +1,31 @@
+package lesson03.conditions;
+/*
+Task 3 (LuckyTicket)
+Определить, является ли билетик ticket счастливым. Номер билетика - целое 6-значное число.
+Счастливый - это такой билетик, у которого сумма старших 3-х цифр и младших 3-х цифр совпадают.
+Ограничения: ticket = 0 … 999999.
+boolean isLuckyTicket(int ticket)
+Входные аргументы int ticket
+Тип результата boolean
+*/
+
+import utils.errorhandling.ExceptionHandling;
+
+public class LuckyTicket {
+    public static void main (String[] args) {
+        System.out.println(isLuckyTicket(123321));
+    }
+
+    public static boolean isLuckyTicket(int ticket){
+        int firstPartOfTicketNumber = ticket / 100000 + ticket % 100000 / 10000 + ticket % 10000 / 1000;
```

Comment:
I do not get what it is when read that.
is it a sum of the first digits on the left side?
I calculated and got 6.88631, is it expected?

Reply:
clas has been rewritten according to a new logic